.github/PULL_REQUEST_TEMPLATE/control_plane_template.md

```diff
@@ -39,10 +39,8 @@ If the automation determines you have breaking changes, i.e. Step 1 from the dia
 [1] [ARM review queue] (for **merge** queues, see [4])  
 The PRs are processed by time opened, ascending. Your PR may show up on 2nd or later page. 
 If you addressed Step 1 from the diagram and your PR is not showing up in the queue, ensure the label `ARMChangesRequested` 
-is removed from your PR. This should cause the label `WaitForARMFeedback` to be added.  
-[2] https://aka.ms/azsdk/support/specreview-channel  
-[3] [List of SDK breaking changes approvers] in pinned Teams announcement  
```

Comment:
Ray, is the list of SDK breaking changes approved still correct? Should we continue maintaining it as a pinned Teams announcement or remove it / migrate to https://aka.ms/azsdkdocs ?

Reply:
The pinned post is still valid. We shall update it next time when we roll out the new SDK breaking change review workflow.